contracts/Launchpeg.sol

```diff
@@ -221,15 +221,19 @@ contract Launchpeg is BaseLaunchpeg, ILaunchpeg {
         ) {
             revert Launchpeg__InvalidPercent();
         }
-        if (_auctionDropInterval == 0) {
+
+        auctionSaleDuration = _mintlistStartTime - _auctionSaleStartTime;
+        if (
+            _auctionDropInterval == 0 ||
+            _auctionDropInterval > auctionSaleDuration / 4
```

Comment:
Add a comment explaining why we revert if auctionDropInterval is greater than a quarter of the auctionSaleDuration

Reply:
I'd just say something like.. "Ensure auction drop interval is not too high by enforcing it is at most 1/4 of the auction sale duration" or something like that.